src/components/formBuilder/FieldTemplate.tsx

```diff
@@ -32,12 +32,15 @@ interface FormPreviewFieldTemplateProps extends FieldTemplateProps {
   setActiveField: SetActiveField;
 }
 
+// eslint-disable-next-line local-rules/persistBackgroundData -- why is this rule triggering here?
```

Comment:
@fregante seems to be a problem with the local rule (possibly just the configuration in .eslintrc.js?)

Reply:
Oh, the file is in `persistBackgroundData.txt`. Why would this file be needed by the background script? I'm guessin g we're importing an exported constant or something.

Module level would still be safe here. We're just preventing unnecessary re-renders because of referential changes between renders.

Reply:
```suggestion
// eslint-disable-next-line local-rules/persistBackgroundData -- Static
```

1. The background page is importing this file indirectly, maybe via the registry
2. `persistBackgroundData.txt` is a list of all of the files that `dist/background.js` ends up bundling, whether intentional or not
3. It's a very common situation and I've been using "Static" or "Unused by the background" as a justification

If it happens often, we might want to add `as const` to the exclusions in the rule, but let's see how it works out in the end.